Sources/Logging/Logging.swift

```diff
@@ -235,7 +235,7 @@ extension Logger {
     ///    - message: The message to be logged. `message` can be used with any string interpolation literal.
     ///    - metadata: One-off metadata to attach to this log message
     ///    - file: The file this log message originates from (there's usually no need to pass it explicitly as it
-    ///            defaults to `#file`).
+    ///            defaults to `#file` (until Swift 5.2) / `#filePath` (from Swift 5.3)).
```

Comment:
Comment is semi true I guess... Would it be better to remove it? It's not like _we_ make that `if ...` but it's just Swift's behavior, so why should _we_ document that here if observed behavior is the same?

Reply:
good catch, thank you!